ci-automation/test.sh

```diff
@@ -176,9 +176,10 @@ function test_run() {
         echo "########### All re-runs exhausted ($retries). Giving up. ###########"
     fi
 
-    # TODO: publish to bincache?
-    # "${tests_dir}/"*.tap
-    # "${tests_dir}/_kola_temp.tar.xz"
-
+    # publish TAP files to buold cache
+    copy_to_buildcache "testing/${vernum}/${arch}/${image}" \
+        "${tests_dir}/"*.tap
+    copy_to_buildcache "testing/${vernum}/${arch}/${image}/debug" \
```

Comment:
nit: why not just put it next to the tap file, skipping the debug subdir?.

Reply:
I would test for existence first before doing the call because it would fail.

Reply:
`copy_to_buildcache` uses `rsync` [under the hood](https://github.com/flatcar-linux/scripts/blob/main/ci-automation/ci_automation_common.sh#L127), so existing directories would be extended, and existing files would be replaced with newer versions where applicable.

Reply:
The *.tap and _kola_temp/* may not match any files and will cause rsync to fail

Reply:
Ah, gotcha. In that case, `kola` did not run at all, and the script would have aborted earlier (since we use `set -euo pipefail`).

Reply:
Yes, was talking about that. In this case we would still fail here because rsync does not find a file under the unexpanded name, so no problem with having it missing.